scala/ppml/src/test/scala/com/intel/analytics/bigdl/ppml/attestation/GramineQuoteGeneratorImplSpec.scala

```diff
@@ -17,10 +17,44 @@
 package com.intel.analytics.bigdl.ppml.attestation
 
 import org.scalatest.{FlatSpec, Matchers}
+import java.io.{FileOutputStream,FileInputStream,BufferedInputStream,BufferedOutputStream}
+import scala.io.Source
 
 class GramineQuoteGeneratorImplSpec  extends FlatSpec with Matchers {
+   
+  //get the envFlag from the 'env-config.txt' file .
+  val envConfigPath=this.getClass.getClassLoader.getResource("attestation/gramine/env-config.txt").getPath
+  val envFlag = Source.fromFile(envConfigPath).getLines.mkString
 
-  // TODO  GramineQuoteGeneratorImplSpec
+  //GramineQuoteGeneratorImplSpec
   "Gramine get Quote " should "work" in {
+    if (envFlag=="TEE") {
```

Comment:
We can use env instead of config in this test. https://github.com/intel-analytics/BigDL/blob/4030121bf12cd31b645c1b9b59b061366c9dcbd7/scala/orca/src/test/scala/com/intel/analytics/bigdl/orca/inference/OpenVINOModelSuite.scala#L39

Reply:
Thanks for the comments, I will update the PR to reflect the change

Reply:
- Updates
- (1) geting `envFlag` from enviroments following this example code.
- (2) all code style problems fixed.